Jose-Paquian.txt

```diff
@@ -0,0 +1,47 @@
+https://codepen.io/alostnight/pen/VwYLNYd?editors=1100
+1. What is Semantic HTML? The tags that are there to give the HTML code meaning.
+    2. What is HTML used for? Mostly web development and web pages.
+    3. What is an attribute and where do we put it? It used on an element with a specific value and it can be used in CSS.
+    4. What is the h1 tag used for? How many times should I use it on a page? It is used for the title of the page and should only be used once.
+    5. Name two tags that have required attributes. the element and class.
+    6. What do we put in the head of our HTML document? <header>
+    7. What is an id? it is a selector that is used when a class isn't enough.
```

Comment:
Remember, it can only be used once per document. Used for very specific targeting